src/main/java/com/minecolonies/api/eventbus/events/colony/AbstractColonyEvent.java

```diff
@@ -1,37 +1,36 @@
-package com.minecolonies.api.colony.event;
-
-import com.minecolonies.api.colony.IColony;
-import net.minecraftforge.eventbus.api.Event;
-import org.jetbrains.annotations.NotNull;
-
-/**
- * This is a colony-related event in the Forge sense, not in the
- * {@link com.minecolonies.api.colony.colonyEvents.IColonyEvent} sense.
- */
-public abstract class AbstractColonyEvent extends Event
-{
-    /**
-     * The colony this event was called in.
-     */
-    @NotNull
-    private final IColony colony;
-
-    /**
-     * Constructs a colony-based event.
-     *
-     * @param colony The colony related to the event.
-     */
-    protected AbstractColonyEvent(@NotNull final IColony colony)
-    {
-        this.colony = colony;
-    }
-
-    /**
-     * Gets the colony related to the event.
-     */
-    @NotNull
-    public IColony getColony()
-    {
-        return colony;
-    }
-}
+package com.minecolonies.api.eventbus.events.colony;
+
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.eventbus.events.AbstractEvent;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Any colony related event, provides the target colony the event occurred in.
+ */
+public abstract class AbstractColonyEvent extends AbstractEvent
+{
+    /**
+     * The colony this event was called in.
+     */
+    @NotNull
+    private final IColony colony;
+
+    /**
+     * Constructs a colony-based event.
+     *
+     * @param colony The colony related to the event.
+     */
+    protected AbstractColonyEvent(@NotNull final IColony colony)
+    {
+        this.colony = colony;
+    }
+
+    /**
+     * Gets the colony related to the event.
+     */
+    @NotNull
+    public IColony getColony()
+    {
+        return colony;
+    }
+}
```

Comment:
there are few classes apparently with cr/lf changes

Reply:
These are just weird diffs by Github, it's because the file moved and slightly altered it doesn't know what to do with them.

Reply:
Yeah, but we shouldn't auto format all files we touch, because that makes it impossible to read the dif